resources/templates/design.md

```diff
@@ -0,0 +1,39 @@
+# Component Name
+_Include a short (1-2 sentence) description of the component here and fill out the following sections as needed_
+
+## Usage
+_(Required)
+Informs readers about how and when this pattern is used including any variations
```

Comment:
Inform readers about when and how this component or pattern should be used, including any variations or related best practices.

Reply:
Much better.